new-frontiers-14/frontier-station-14
# Salvage Outpost Dungeon Theme Backrooms-Fix

## About the PR
Room pool for generator was missing 2 rooms. This PR fixes that.

## Why / Balance
Fixes

## How to test
1. Create map space with command `mapping #mapID` (ex.: `mapping 111`).
2. Initialize created mapping space with command `mapinit #mapID` (ex.: `mapinit 111`).
3. Run `dungen #mapID #Theme #x #y` command for `SalvageOutpost` template (ex.: `dungen 111 SalvageOutpost 0 0 `).
4. Observe lack of backrooms (empty rooms with yellow floors).

## Media

## Breaking changes
none afaik

**Changelog**
:cl: erhardsteinhauer
- fix: Fixed backrooms on salvage outpost dungeon theme.
